backend/Testing/Fixtures/IntegrationFixture.cs

```diff
@@ -0,0 +1,77 @@
+using System.IO.Compression;
+using System.Net;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using LexCore.Utils;
+using Shouldly;
+using Testing.ApiTests;
+using Testing.Services;
+using TusDotNetClient;
+using static Testing.Services.Constants;
+
+namespace Testing.Fixtures;
+
+public class IntegrationFixture : IAsyncLifetime
+{
+    private readonly string _templateRepoName = "test-template-repo.zip";
+    public FileInfo TemplateRepoZip { get; } = new(Path.Join(BasePath, "_template-repo_.zip"));
+    public DirectoryInfo TemplateRepo { get; } = new(Path.Join(BasePath, "_template-repo_"));
+    public ApiTestBase AdminApiTester { get; } = new();
+    private string AdminJwt = string.Empty;
+
+    public async Task InitializeAsync()
+    {
+        DeletePreviousTestFiles();
+        Directory.CreateDirectory(BasePath);
+        InitTemplateRepo();
+        AdminJwt = await AdminApiTester.LoginAs(AdminAuth.Username, AdminAuth.Password);
+    }
+
+    public Task DisposeAsync()
+    {
+        return Task.CompletedTask;
+    }
+
+    private static void DeletePreviousTestFiles()
+    {
+        if (Directory.Exists(BasePath)) Directory.Delete(BasePath, true);
+    }
+
+    private void InitTemplateRepo()
+    {
+        var assembly = Assembly.GetExecutingAssembly();
+        var templatePath = Path.Join(Path.GetDirectoryName(assembly.Location), _templateRepoName);
+        File.Copy(templatePath, TemplateRepoZip.FullName);
```

Comment:
there's probably no need to copy the zip anywhere, we can just change `TemplateRepoZip` to point at it's location from the assembly.

Reply:
The zip is used in FinishLexboxProjectResetWithRepo, but I don't think it's ever modified. So yeah, we could just point at its location from the assembly. I'll do that.

Reply:
Done in commit b89fd858, but I'm not 100% happy with how long the source line for TemplateRepoZip turned to be. I'll split it into two lines in another commit.

Reply:
Done in commit 900c10b2, which I'm a lot happier with.